extra/resources/ClusterMon

```diff
@@ -153,7 +153,8 @@ ClusterMon_monitor() {
     if [ -f $OCF_RESKEY_pidfile ]; then
 	pid=`cat $OCF_RESKEY_pidfile`
 	if [ ! -z $pid ]; then
-	    kill -s 0 $pid >/dev/null 2>&1; rc=$?
+	    ps -wwfp $pid | grep $OCF_RESKEY_pidfile >/dev/null 2>&1 && \
+	        kill -s 0 $pid >/dev/null 2>&1; rc=$?
```

Comment:
+ as I mentioned at https://github.com/ClusterLabs/resource-agents/pull/846#issuecomment-250114916, it's idiomatic to use `grep -q` instead of redirections (`ps`, on the other hand, would be eligible for `stderr` suppression).